modules/declarative-configuration-auth-resources.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-role-based-access-control-3630.adoc
+:_content-type: CONCEPT
+[id="declarative-configuration-auth-resources_{context}"]
+= Declarative configuration for authentication and authorization resources
+
+[role="_abstract"]
+You can use declarative configuration for authentication and authorization resources such as authentication providers, roles, permission sets, and access scopes. With declarative configuration, you can update configurations by storing them in files in repositories and apply them to the system. Declarative configuration is useful, for example, if you are using a GitOps workflow.
+
+To use declarative configuration, you create YAML files that contain configuration information about authentication and authorization resources. These files, or configurations, are added to {product-title-short} by using a mount point during Central installation. The configuration files are stored in config maps or secrets, depending on the type of resource. Store configurations for authentication providers in a secret for greater security. You can store other configurations in config maps.
+
+A single config map or secret can contain more than one configuration of multiple resource types. This allows you to limit the number of volume mounts for the Central instance.
+
+[id="restrictions-declarative-config-resources"]
+== Restrictions for resources created from declarative configuration
+
+Because resources can reference other resources (for example, a role can reference both a permission set and access scope), the following restrictions for references apply:
+
+* A declarative configuration can only reference a resource that is either also created declaratively or a default {product-title-short} resource; for example, a resource such as a default role, permission set, or access scope.
```

Comment:
nit: we've made the change in 4.0 to call those resources out in the UI as `System`. Might be best to reuse the same language here, unless its already been used in a different context?

Sample screenshot for the roles:
![Screenshot 2023-06-26 at 05 22 32](https://github.com/openshift/openshift-docs/assets/89904305/50775b31-4b5d-4e00-924c-ff6a16199de9)